src/transformers/modeling_auto.py

```diff
@@ -417,6 +423,7 @@
         (ElectraConfig, ElectraForQuestionAnswering),
         (ReformerConfig, ReformerForQuestionAnswering),
         (FunnelConfig, FunnelForQuestionAnswering),
+        (LxmertConfig, LxmertForQuestionAnswering),
```

Comment:
was missing

Reply:
We definitely need some kind of script to check those automatically ;-)

Reply:
Yes, we do!